src/ObjectUtils/ElfFile.cpp

```diff
@@ -432,6 +435,21 @@ ErrorMessageOr<void> ElfFileImpl<ElfT>::InitProgramHeaders() {
     return ErrorMessage(std::move(error));
   }
 
+  // Compute the image size as the difference between the end address of the last loadable segment
+  // and the start address of the first loadable segment.
+  std::optional<uint64_t> first_loadable_segment_vaddr;
```

Comment:
As discussed, I would suggest one of two options:
- Get rid of GetImageSize() for Elf until we actually need it. 
- Add a clear comment that indicates that the choice of what image size is in the Elf case is a choice that can be changed if something else is needed (e.g. sum of sizes of loadable segments).